### ISSUE TITLE
`interactive` ignore alternate C2 flags

### ISSUE THREAD

**Issue by moloch--** [BishopFox/sliver#1181]
The `interactive` command re-uses the compile time C2 endpoints instead of the ones provided by the command.

**Comment by RafBishopFox** [comment]
I put a fix together in 85f51a8752b40596f00f3a77c494ae55d17f08de , but it only works if you want to use a transport that was compiled in. For example, if you want to use a different HTTP(S) C2, that works in my testing with the fix.

However, if you want to switch to a different transport that was not compiled in initially, I think we would have to make it so that all transports are compiled in regardless of the transports specified at compile time. I am not sure if that is a change we want to make, so I wanted to get your thoughts on that first.

**Comment by rkervella** [comment]
@RafBishopFox that's the acceptable tradeoff for the moment. If/When we eventually get the transports ported to WASM modules, we'll be able to inject them at runtime.

**Comment by r00t0v3rr1d3** [comment]
Yup. That was the previously known limitation. Thank you guys for the bug fix. I will test it next week.
